src/RubberDuckyParking/parking/views.py

```diff
@@ -14,9 +14,13 @@ def host_index(request):
     return render(request, 'parking/host_index.html', context)
 
 
-def host_detail(request, host_id="empty"):
-    context = {"host_id": host_id}
-    return render(request, 'parking/host_details.html', context)
+def host_detail(request):
+    if request.user.is_authenticated:
```

Comment:
Will request.user cause problems now that the user is an abstract class?

Reply:
nope, it works on my local. It seems like your db might be messed up as far as users go. I can run you through it because sometimes it can be tricky, but you'll probably need to flush your db and re run migrations